62.md

```diff
@@ -0,0 +1,61 @@
+NIP-62
+======
+
+Request to Vanish
+-----------------
+
+`draft` `optional`
+
+This NIP offers a Nostr-native way to request a complete reset of a key's fingerprint on the web. This procedure is legally binding in some jurisdictions, and thus, supporters of this NIP should truly delete events from their database. 
+
+## Request to Vanish from Relay
+
+Kind `62` requests a specific relay to delete everything, including [NIP-09](09.md) Deletion Events, from the `.pubkey` until its `.created_at`. 
+
+```jsonc
+{
+  "kind": 62,
+  "pubkey": <32-byte hex-encoded public key of the event creator>,
+  "tags": [
+    ["relay", "<relay url>"]
+  ],
+  "content": "<reason or note>",
+  //...other fields
+}
+```
+
+The tag list MUST include at least one `relay` value. 
+
+Content MAY include a reason or a legal notice to the relay operator. 
+
+Relays MUST fully delete any events from the `.pubkey` if their service URL is tagged in the event. 
+
+Relays SHOULD delete all [NIP-59](59.md) Gift Wraps that p-tagged the `.pubkey`, deleting all DMs to the pubkey.
```

Comment:
1. Should this also mention the deprecated `kind:4`? It's basically the same thing right?
2. This wording implies tagged gift wraps can be deleted even if their service URL is not tagged in the event (this is mentioned only for deleting by `pubkey`). Is this intended? If so, why?

Reply:
No, kind:4s that were sent to you are owned by the sender. GiftWraps are not owned by anyone (random signer).

Reply:
adjusted the wording for DMs